api/apitest/apitest.go

```diff
@@ -42,22 +42,23 @@ func DefaultTextileConfig(t util.TestingTWithCleanup) core.Config {
 	require.NoError(t, err)
 
 	return core.Config{
-		Hub:                       true,
-		Debug:                     true,
-		AddrAPI:                   util.MustParseAddr(fmt.Sprintf("/ip4/127.0.0.1/tcp/%d", apiPort)),
-		AddrAPIProxy:              util.MustParseAddr("/ip4/0.0.0.0/tcp/0"),
-		AddrMongoURI:              GetMongoUri(),
-		AddrMongoName:             util.MakeToken(12),
-		AddrThreadsHost:           util.MustParseAddr("/ip4/0.0.0.0/tcp/0"),
-		AddrIPFSAPI:               GetIPFSApiAddr(),
-		AddrGatewayHost:           util.MustParseAddr(fmt.Sprintf("/ip4/127.0.0.1/tcp/%d", gatewayPort)),
-		AddrGatewayURL:            fmt.Sprintf("http://127.0.0.1:%d", gatewayPort),
-		IPNSRepublishSchedule:     "0 1 * * *",
-		CustomerioAPIKey:          os.Getenv("CUSTOMERIO_API_KEY"),
-		CustomerioConfirmTmpl:     os.Getenv("CUSTOMERIO_CONFIRM_TMPL"),
-		CustomerioInviteTmpl:      os.Getenv("CUSTOMERIO_INVITE_TMPL"),
-		EmailSessionSecret:        SessionSecret,
-		MaxBucketArchiveRepFactor: 4,
+		Hub:                        true,
+		Debug:                      true,
+		AddrAPI:                    util.MustParseAddr(fmt.Sprintf("/ip4/127.0.0.1/tcp/%d", apiPort)),
+		AddrAPIProxy:               util.MustParseAddr("/ip4/0.0.0.0/tcp/0"),
+		AddrMongoURI:               GetMongoUri(),
+		AddrMongoName:              util.MakeToken(12),
+		AddrThreadsHost:            util.MustParseAddr("/ip4/0.0.0.0/tcp/0"),
+		AddrIPFSAPI:                GetIPFSApiAddr(),
+		AddrGatewayHost:            util.MustParseAddr(fmt.Sprintf("/ip4/127.0.0.1/tcp/%d", gatewayPort)),
+		AddrGatewayURL:             fmt.Sprintf("http://127.0.0.1:%d", gatewayPort),
+		IPNSRepublishSchedule:      "0 1 * * *",
+		MaxRepublishingConcurrency: 5,
```

Comment:
Can we make this match the other related param, like `IPNSRepublishConcurrency`?

Reply:
done